README.adoc

```diff
@@ -88,9 +91,13 @@ You can then activate the profile using `-Pspanner` such as `mvn compile -Pspann
 
 == User Guide
 
-The following section describes recommendations and best practices for how to best use the Cloud Spanner Hibernate Dialect.
+This guide contains a variety of best practices for using Hibernate with Spanner which can significantly improve the performance of your application.
+
+=== Schema Creation and Entity Design
 
-=== Use Generated UUIDs for ID Generation
+Hibernate generates statements based on your Hibernate entity's design. Following these practices can result in better DDL and DML statement generation which can improve performance.
+
+==== Use Generated UUIDs for ID Generation
```

Comment:
Do you want to comment on why autogenerated is better than self-created?

Reply:
Done.